system/include/emscripten/eventloop.h

```diff
@@ -26,6 +26,10 @@ void emscripten_set_immediate_loop(EM_BOOL (*cb)(void *userData), void *userData
 long emscripten_set_interval(void (*cb)(void *userData), double intervalMsecs, void *userData);
 void emscripten_clear_interval(long setIntervalId);
 
+void emscripten_runtime_keepalive_push();
+void emscripten_runtime_keepalive_pop();
+EM_BOOL emscripten_runtime_keepalive_check();
```

Comment:
Why does EM_BOOL exist? Why not just use int or bool here?

Reply:
It exists yes.   It pre-dates my involvement with the project.

I guess we could use `<stdbool.h>` these days but I'm not sure I'd want to force all emscripten users to include `<stdbool.h>`.. since it does pollute the global namespace.